docker/docker-tezos-packages.py

```diff
@@ -171,6 +173,11 @@ def validate_dists(dists, versions, target_os):
     """
     )
 
+    with open(os.path.join(args.output_dir, ".artifact_list"), "r") as f:
+        artifacts += f.read().splitlines()
+
+    call(f"rm -rf {os.path.join(args.output_dir, '.artifact_list')}")
```

Comment:
I believe that `-r` should be unnecessary here:
```suggestion
    call(f"rm -f {os.path.join(args.output_dir, '.artifact_list')}")
```